clippy.toml

```diff
@@ -15,7 +15,6 @@ disallowed-methods = [
 ]
 
 disallowed-types = [
-    "rand::OsRng",
```

Comment:
This was mainly for avoiding people creating RNGs on the fly instead of passing them down by parameters, but yeah I'm fine with removing that from this list!